distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1644,7 +1641,7 @@ type Table
     make_constant_column : Any -> Column
     make_constant_column self value =
         if Table_Helpers.is_column value then Error.throw (Illegal_Argument.Error "A constant value may only be created from a scalar, not a Column") else
-            Column.from_vector_repeated value.pretty [value] self.row_count
+            Column.from_repeated_item value.pretty value self.row_count
```

Comment:
Changed yet again back to `pretty` because `to_text` was breaking lots of tests.